src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
```

Comment:
What about `calc(1em * --ms4)`? That's ~ 2.4em and uses our modular scale.

I agree that the `--space-*` variables aren't a wide enough range. We should switch to the same numeric system as our scale for consistency. It goes from `--ms-6` to `--ms6`.